libs/datasets/timeseries.py

```diff
@@ -1134,9 +1145,20 @@ def aggregate_regions(
     aggregate_map: Mapping[Region, Region],
     aggregations: Sequence[StaticWeightedAverageAggregation] = WEIGHTED_AGGREGATIONS,
     *,
-    ignore_na: bool = False,
+    reporting_ratio_required_to_aggregate: Optional[float] = None,
```

Comment:
Why optional instead of float with default of 0.0 or 1.0?

Reply:
I wanted to use none to specify that we're disabling filtering by ratio reported but can change it back